### ISSUE TITLE
Microsoft API Extractor

### ISSUE THREAD

**Issue by elsassph** [Simn/ts2hx#29]
Maybe would be better than maintaining our custom `d.ts` lexer?
https://github.com/Microsoft/web-build-tools/wiki/API-Extractor

**Comment by Simn** [comment]
Yes @nadako brought that up a while ago, but nobody ended up looking into it. I'm more than happy to retire this project.
